core/test_blockchain.go

```diff
@@ -1004,13 +1004,14 @@ func TestReorgReInsert(t *testing.T, create func(db ethdb.Database, chainConfig
 // Insert two different chains that result in the identical state root.
 // Once we accept one of the chains, we insert and accept A3 on top of the shared
 // state root
-//   G   (genesis)
-//  / \
-// A1  B1
-// |   |
-// A2  B2 (A2 and B2 represent two different paths to the identical state trie)
-// |
-// A3
+//
+//	  G   (genesis)
+//	/  \
+//	A1  B1
+//	|   |
+//	A2  B2 (A2 and B2 represent two different paths to the identical state trie)
+//	|
```

Comment:
can we fix this one up a little bit by spacing the A1 and B1 column a little bit more so that they are right below the `/` and `\` chars?

Reply:
thanks

Reply:
done thanks